react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,22 @@
+import Topics from './Topics';
+import { useState } from 'react';
+
+export default function Accordion({ topics }) {
+  const [clicked, setClicked] = useState('');
```

Comment:
The default value for `clicked` shouldn't be an empty string since the type that it holds after a click is a Topic. You should keep the data type of the state variables consistent.

Since Topic is an object, the way to indicate absence of an object is to use `undefined`. So that should be your default value.